Docs/source/theory/kinetic_fluid_hybrid_model.rst

```diff
@@ -1,7 +1,7 @@
 .. _theory-kinetic-fluid-hybrid-model:
 
-Kinetic-fluid Hybrid Model
-==========================
+Hybrid Model based on Maxwell-Ampere coupled with Ohm's law
+===========================================================
```

Comment:
Yeah, I agree with you that mentioning the field equations would be good.
I wonder, though, if we should change this as follows since the "hybrid" part is meant to signify that it is part fluid part kinetic (so saying hybrid without "kinetic-fluid" is maybe not clear)
```suggestion
Maxwell-Ampere coupled with Ohm's law
=====================================
```